source/includes/fact-favorite-query.rst

```diff
@@ -0,0 +1,26 @@
+You can favorite a query so that you can find it later. If you
+load a favorited query, you can change it without
+changing the original.
```

Comment:
Maybe replace "original" with "saved version" ?

EDIT: I see you used "original saved copy" in source/includes/fact-save-pipeline.rst. Let's use that here as well.